sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientTest.java

```diff
@@ -676,11 +692,12 @@ public void recognizeCustomFormLabeledDataWithJpgContentType(HttpClient httpClie
                         .getSyncPoller();
                 trainingPoller.waitForCompletion();
 
-                SyncPoller<FormRecognizerOperationResult, List<RecognizedForm>> syncPoller = client.beginRecognizeCustomForms(
-                    trainingPoller.getFinalResult().getModelId(), toFluxByteBuffer(data), dataLength,
-                    new RecognizeCustomFormsOptions()
-                        .setContentType(FormContentType.IMAGE_JPEG)
-                        .setPollInterval(durationTestMode)).getSyncPoller();
+                SyncPoller<FormRecognizerOperationResult, List<RecognizedForm>> syncPoller =
```

Comment:
Is there a reason why so many format changes?

Reply:
No good reason, really. Had imported some recent formatting changes that should have been applied.